src/liblinux/BMGUtils.c

```diff
@@ -28,8 +28,8 @@
 #include <stdlib.h>
 
 #include "BMGUtils.h"
-#include "osal_preproc.h"
-#include "liblinux/BMGImage.h"
+#include "../osal_preproc.h"
+#include "BMGImage.h"
```

Comment:
This change is consistent with declarations elsewhere, e.g.
https://github.com/mupen64plus/mupen64plus-video-rice/blob/master/src/liblinux/BMGImage.c#L32
https://github.com/mupen64plus/mupen64plus-video-rice/blob/master/src/liblinux/bmp.c#L34
https://github.com/mupen64plus/mupen64plus-video-rice/blob/master/src/liblinux/pngrw.c#L33